src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -56769,6 +56769,80 @@ static void F(object? x)
                 Diagnostic(ErrorCode.WRN_ConvertingNullableToNonNullable, "(object)x").WithLocation(8, 13));
         }
 
+        [Fact]
+        public void InheritNullabilityOfNonNullableClassMember()
+        {
+            var source =
+@"#pragma warning disable 8618
+class C<T>
+{
+    internal T F;
+}
+class Program
+{
+    static void F0(string? s)
+    {
+        var a1 = new C<string>() { F = s };
+        _ = a1.F/*T:string?*/;
```

Comment:
>_ = a1.F/*T:string?*/; [](start = 8, length = 22)

Please add a an attempt to use this field in an unsafe manner. The goal of the fix is to modify observable behavior of the compiler.